tests/Gedmo/Tree/NestedTreeRootRepositoryTest.php

```diff
@@ -157,7 +159,7 @@ public function testShouldSupportChildrenHierarchyAsHtml(): void
         $childOpen = '';
         $childClose = '';
         $nodeDecorator = static function ($node) {
-            return str_repeat('-', $node['level']).$node['title']."\n";
+            return str_repeat('-', $node['level'] - 1).$node['title']."\n";
```

Comment:
is this change needed because the level of `RootCategory` has been changed from 0 to 1? if that's the case I think we should change the expected output instead of this or do not change `RootCategory` level and create another fixture to test this new feature.

Reply:
Yes, that's the case. Changing the root level to 1 would change the test output from:
```
-Fruits
-Vegitables
--Carrots
--Potatoes
```
to:
```
--Fruits
--Vegitables
---Carrots
---Potatoes
```
I decided to change the node decorator to keep the old output as it looks more sensible. Other than that, I don't think it really matters if I changed the output instead. But, if you prefer that, I can change the output.
As for creating another fixture, I agree that would be ideal, but I've had to change the expected level in multiple tests that use the RootCategory fixture and if I had to to another fixture, this would result in a lot of duplicated tests. It just seemed a bit too much. If that's the way you prefer to go though, I'll need some time to work on it.

Reply:
Just to give you some context on this feature and the point of it - it was necessary because we were incrementally migrating from another framework, working with a legacy DB that we couldn't change at the time.

Reply:
It's not a blocker, I was just a bit concerned about someone reading the code and do not understand that that `-1` inside the method is because `RootCategory` has level 1, I guess is like in the other changes it would be:

```diff
- static::assertSame(1, $node->getLevel());
+ static::assertSame(1, $node->getLevel() - 1);
```
Anyway, it's fine by me when it's rebased.